packages/api/endpoints/granules.js

```diff
@@ -648,14 +666,15 @@ async function bulkReingest(req, res) {
       type: 'BULK_GRANULE_REINGEST',
       envVars: {
```

Comment:
What is the type of `envVars`? I'd guess that it's probably env vars that should be set in some other process. If that's the case, then the type is probably `{ [k: string]: string }`. We're using `false` below, which should probably be `'false'` in that case.

Reply:
Agreed - and particularly in the case of KNEX_DEBUG, as 

```

export const isKnexDebugEnabled = (
  env: NodeJS.ProcessEnv = {}
) => env.KNEX_DEBUG === 'true';
```